example/examples/Injection.tsx

```diff
@@ -102,6 +102,11 @@ export default class Injection extends Component<Props, State> {
                 window.self.document.body.style.backgroundColor = "cyan";
               }
 
+              // Example usage of injectedJavaScriptObject({hello: 'world'})
+              const injectedObjectJson = window.ReactNativeWebView.injectedObjectJson();
+              const injectedObject = JSON.parse(injectedObjectJson);
+              console.log(injectedObject); // Should be: { hello: 'world' }
```

Comment:
At the very least, I think this should include the text "injectedObject" or "injectedJavaScriptObject"

But I think it would be _ideal_ if you could actually put the content into the webview using the `appendChild` technique from further down in this script, like with `namedFramesAtAfterContentLoadedEle`?